src/components/Posts/Cards/SkeletonPostPreviewCard/SkeletonPostPreviewCard.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import { SkeletonPostPreviewCard } from './SkeletonPostPreviewCard';
+
+beforeEach(() =>
+  render(
+    <MemoryRouter>
+      <SkeletonPostPreviewCard />
+    </MemoryRouter>,
+  ),
+);
+
+describe('SkeletonPostPreviewCard', () => {
+  it('renders the component', () => {
+    expect(screen.getByTestId('skeleton')).toBeInTheDocument();
```

Comment:
I don't think these tests make sense. You render a component and check if the root of it has rendered.
Probability of this test to fail is approaching 0

It would be better to at least do a snapshot test